docs/source/workflows/data_sources.rst

```diff
@@ -15,15 +15,21 @@ Uploading a new file with the same name will produce a new version of the file w
 The columns in the CSV are extracted and parsed by a mapping of column header names to user-created descriptors.
 Columns in the CSV that are not mapped with a descriptor are ignored.
 
+An optional list of identifiers can be specified.
```

Comment:
"identifiers" feels a bit ambiguous here. From this paragraph I'm not sure how that parameter is used; what exactly is it identifying? Is it to specify a list of unique keys in the CSV, or in something downstream of the CSV ingest?

Reply:
Identifiers are a bit ambiguous because they're not useful with simple mixture support. I'm in the process of adding more documentation in the simple mixture PR (#324).